data/raw/combine_updated_gene_file.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+"""
+This script reads three gene information files and combine them into a
+single tab-delimited file, which will be used as the input file of the
+management command `update_gene_names_aliases.py` in the parent directory.
+"""
+
+import csv
+
+gene_annotations = dict()
+
+
+def get_PA14_names():
+    """
+    Reads in a CSV file that maps PAO1 names (aka. systematic_name) to PA14
+    names. If a line's column #4 is "Pseudomonas aeruginosa UCBPP-PA14", then
+    column #2 is PAO1 name, and column #5 is its PA14 name.
+
+    Returns a dict whose key is PAO1 name, and value is a list of PA14 names.
+    """
+
+    PA14_src = 'Pseudomonas_aeruginosa_PAO1_107_orthologs.csv'
+    pao1_to_pa14 = dict()
+    with open(PA14_src) as fh:
+        reader = csv.reader(fh, delimiter=',', quotechar='"')
+        line_num = 0
+        for row in reader:
+            line_num += 1
+            if line_num == 1 or row[3] != 'Pseudomonas aeruginosa UCBPP-PA14':
+                continue
+
+            pao1_name = row[1]
+            pa14_name = row[4]
```

Comment:
I always try to avoid hardcoding column indexes. Always best if you can refer to column by name... possibly csv.DictReader could be used here?

Reply:
In general, it is a good idea to use column names instead of column numbers. But the input files that were downloaded online and saved in `raw` are very preliminary (some column names even include typo), so I used column numbers here. I will add some detailed comments to list the column names in each file.